sources/lib/api/gcp/bigquery/_udf.py

```diff
@@ -13,34 +13,27 @@
 """Google Cloud Platform library - BigQuery UDF Functionality."""
 
 import json
+import gcp.storage
 
 
 class FunctionCall(object):
```

Comment:
Do we still need this as well as FunctionEvaluation?
I am assuming we're no longer going to allow you to locally call a UDF directly.